lib/app/widgets/main_scaffold.dart

```diff
@@ -1,7 +1,9 @@
 import 'package:flutter/material.dart' hide Route;
+import 'package:flutter/material.dart' as prefix0;
```

Comment:
Ugh this looks like an artifact of https://github.com/dart-lang/sdk/issues/36890